collection-scripts/gather_kube_apiserver_termination_logs

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+# Downloads the apiserver watch termination logs (and its rotated copies) from
+# /var/logs/kube-apiserver/termination.log on each master node.
+BASE_COLLECTION_PATH="${BASE_COLLECTION_PATH:-/must-gather}"
+
+oc adm node-logs --role=master --path="kube-apiserver/termination.log" > ${BASE_COLLECTION_PATH}/apiserver_watch_termination.log
```

Comment:
question: this properly separates the three files?

Reply:
it stores them into one file.

Reply:
I'll tweak it to split them up.

Reply:
for some reason gather_audit_logs were disabled... I have re-enabled them for must gather and it does include the termination.log